mobile/native/src/ln_dlc/mod.rs

```diff
@@ -264,6 +264,7 @@ pub fn run(data_dir: String, seed_dir: String, runtime: &Runtime) -> Result<()>
             config::get_oracle_info().into(),
         )?;
         let node = Arc::new(node);
+        tracing::info!(node_id = node.info.pubkey.to_hex(), "Node started");
```

Comment:
🔧 I think this log is redundant: https://github.com/get10101/10101/blob/ca30d2bde4dfe19f414a16967c8fcc23e0b6232b/crates/ln-dlc-node/src/node/mod.rs#L487.